src/main/scala/sbtghactions/GenerativePlugin.scala

```diff
@@ -438,11 +437,13 @@ on:
     branches: [${branches.map(wrap).mkString(", ")}]$renderedTags
 
 ${renderedEnv}jobs:
-${indent(jobs.map(compileJob(_, sbt)).mkString("\n\n"), 1)}"""
+${indent(jobs.map(compileJob(_, sbt)).mkString("\n\n"), 1)}
+"""
```

Comment:
Had to add a trailing newline here because of the change to `IO.write` (it doesn't automatically do that).